app/models/facility.rb

```diff
@@ -234,4 +231,39 @@ def localized_facility_size
     return unless facility_size
     I18n.t("activerecord.facility.facility_size.#{facility_size}", default: facility_size.capitalize)
   end
+
+  def locale
+    LOCALE_MAP.dig(country.downcase, state.downcase) || LOCALE_MAP.dig(country.downcase, "default") || LOCALE_MAP["default"]
+  end
+
+  LOCALE_MAP = {
```

Comment:
@dburka I just want to run this by you. This is based on the info I got from Praveen. It's worth noting that there may be some pre-optimizations. Some of the states in Ethiopia listed here do not yet have facilities, but when we add facilities, messages for their patients will be localized. The localization protocol is in the method above:
first we try to localize by country and state; if the state isn't found, we use the country's default language; if the country isn't found, we use English. 

Does that sound right to you? I'm not sure what the process for adding a new country is like, but it might be worth thinking about how rollout to new countries should be handled.

Reply:
It sounds good. You're right that we should definitely have a process in place not to miss setting a default language per country. For instance, if Sri Lanka starts soon.